README.md

```diff
@@ -90,7 +90,7 @@ Attribute                          | Description
 ```languageValuesOverridePathMap```    | (Since 2.2.0) (Optional) Map of `language_code:path` entries that you want to override the default language values folder with. Defaults to empty map.
 ```minimumTranslationPercentage``` | (Since 2.3.0) (Optional) The minimum accepted percentage of translated strings per language. Languages with fewer translated strings will not be fetched. Defaults to no minimum, allowing all languages to be fetched.
 ```filters```                      | (Since 2.4.0) (Optional) List of PoEditor filters to use during download. Defaults to empty list. Accepted values are defined by the POEditor API.
-
+```resFileName``` | (Since 3.1.0) (Optional) Sets the file name for the imported string resource xml files. Defaults to `strings`
```

Comment:
Minor typo
```suggestion
```resFileName``` | (Since 3.1.0) (Optional) Sets the file name for the imported string resource XML files. Defaults to `strings`
```

Reply:
Fixed!